README.md

```diff
@@ -13,3 +13,6 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
```

Comment:
asdasd

__
[![sema-logo](http://localhost:3000/img/sema-tray-logo.gif)](http://localhost:3000/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Fault-tolerant, Maintainable